content/en/docs/tasks/debug/overview.md

```diff
@@ -3,17 +3,19 @@ reviewers:
 - brendandburns
 - davidopp
 content_type: concept
-title: Troubleshooting
+title: Debugging overview
+description: General overview and how to get help.
+weight: 10
 ---
 
 <!-- overview -->
```

Comment:
I'm not keen on the new page URL and title. I would merge this content into Monitoring, Logging, and Debugging (`content/en/docs/tasks/debug/_index.md`).